Astro 12/20 release notes (#1664)
astronomer/docs: astro/release-notes.md

**Latest CLI Version**: 1.8.4 ([Release notes](cli/release-notes.md))

## December 20, 2022 

### Additional improvements 

- Workspace Members can now view **Workspace settings** in the Cloud UI.

> 

Reviewer: Added OneLogin and Ping Identity (SAML) support to Astro supported IdPs

**Latest CLI Version**: 1.8.4 ([Release notes](cli/release-notes.md))

## December 20, 2022 

### Additional improvements 

- You can now configure OneLogin and Ping Identity as identity providers on Astro. See [Configure your identity provider](configure-idp.md#configure-your-identity-provider).

> - Workspace Members can now view **Workspace settings** in the Cloud UI.

Reviewer 2: ```suggestion
- Workspace Members can now view **Workspace settings** in the Cloud UI.
- When node groups are collapsed in the lineage graph in the Cloud UI, they now show only the total number of connected **Jobs** and **Datasets** instead of showing each job and dataset. This makes the lineage graph easier to navigate.

    ![Collapsed node in lineage graph of Cloud UI](/img/release-notes/collapsed-lineage-node.png)
```

**Latest CLI Version**: 1.8.4 ([Release notes](cli/release-notes.md))

## December 20, 2022 

### Additional improvements 

- You can now configure OneLogin and Ping Identity as identity providers on Astro. See [Configure your identity provider](configure-idp.md#configure-your-identity-provider).
- Workspace Members can now view **Workspace settings** in the Cloud UI.

> - When node groups are collapsed in the lineage graph in the Cloud UI, they now show only the total number of connected **Jobs** and **Datasets** instead of showing each job and dataset. This makes the lineage graph easier to navigate.

Reviewer 2: ```suggestion
- Node groups that are collapsed in the lineage graph in the Cloud UI now show only the total number of connected **Jobs** and **Datasets**, instead of listing each job and dataset. This makes the lineage graph easier to navigate.
```

>     ![Collapsed node in lineage graph of Cloud UI](/img/release-notes/collapsed-lineage-node.png)

Author: ```suggestion
```
